docs/cluster-management/configure-api.md

```diff
@@ -253,6 +253,37 @@ scm:
         oauthClientSecret: YOUR-APP-SECRET
 ```
 
+### Notifications Plugin
+
+Currently, only an email notifications plugin is supported. Additional notifications plugins may be added in the future.
+
+Additional notifications plugins can be configured using the following format:
+
+```yaml
+# config/local.yaml
+notifications:
+    email:
+        host: smtp.yourhost.com
+        port: 25
+        from: example@email.com
+    webhook: https://hooks.slack.com/services/T00000000/B00000000/XXXXXXXXXXXX
+```
+
+#### Email Notifications
```

Comment:
I think the "Email Notifications" section should be above the line "currently, only an email ...". Otherwise, putting an almost-same example config seems redundant and confusing